packages/lit-dev-content/src/components/litdev-search.ts

```diff
@@ -68,6 +73,20 @@ function isApiLink(url: string) {
   return url.includes('docs/api/');
 }
 
+/**
+ * Used to mark a search page suggestion with the DOC chip.
+ */
+function isDocsLink(url: string) {
+  return url.includes('docs/') && !isApiLink(url);
```

Comment:
Can we do a more specific condition, like startsWith, or `new URL(url).pathname.startsWith('docs/')`?

Reply:
they are not full URLs but rather `/docs/category/doc-name/index.html` I switched to using `startsWith` LMK what you think

Reply:
lgtm